src/templates/widget_index_case.html

```diff
@@ -16,19 +16,19 @@
     <div class="form-group">
       <div class="col-sm-11">
         <input class="fd-index-property-source form-control"
-               value="<%-v['calculate']%>" type="text" placeholder="Calculation" />
+               value="<%-v['calculate']%>" type="text" placeholder="Referenced Case ID" />
```

Comment:
It's confusing that this gets saved to `'calculate'`. Just verifying that when this makes an index block it makes something like this:
```xml
<index>
     <{relationship identifier} case_type={case_type} relationship={relationship}>{Referenced case ID}</ {relationship identifier}>
</index>
```